File: spread/general/test/hello.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
	if (argc > 1) {
		printf("Hello from C: %s\n", argv[1]);
	} else {
		printf("Hello from C!\n");
	}
	return 0;
}

File: spread/general/test/hello.go
package main

import (
	"fmt"
	"os"
)

func main() {
	if len(os.Args) > 1 {
		fmt.Println("Hello from Go:", os.Args[1])
	} else {
		fmt.Println("Hello from Go!")
	}
	os.Exit(0)
}
